test/format/format_config.c

```diff
@@ -2220,3 +2222,25 @@ config_compact(void)
         config_off(NULL, "ops.compaction");
     }
 }
+
+/*
+ * config_obsolete_cleanup --
+ *     Obsolete cleanup configuration.
+ */
+static void
+config_obsolete_cleanup(void)
+{
+    uint32_t wait_seconds;
+    char confbuf[128];
+
+    if (!config_explicit(NULL, "obsolete_cleanup.method")) {
+        if (mmrand(&g.extra_rnd, 1, 10) < 2)
```

Comment:
Happy to increase this probability. What do you think @kommiharibabu?

Reply:
The `reclaim_space` option is not used in production, so this probability is enough.